e2e/tools/validator/src/validator/cli/__init__.py

```diff
@@ -253,6 +257,17 @@ def report_validation_results(
         report.write(f"  * MAPE: {res.mape}\n")
         report.flush()
 
+        mse_ok = True
+        mape_ok = True
+
+        if v.max_mse is not None and res.mse.error is None and res.mse.value > v.max_mse:
+            click.secho(f"MSE exceeded threshold. mse: {res.mse.value}, max_mse: {v.max_mse}", fg="red")
+            mse_ok = False
```

Comment:
How about we enforce setting a Max MSE and MAPE for all validations? I.e. This field can't be empty

Reply:
in my opinion it is better to keep it optional